packages/cli/src/util/file.ts

```diff
@@ -1,4 +1,4 @@
-import fs, {WriteFileOptions} from "node:fs";
```

Comment:
I do not like it if we have to do `fs.WriteFileOptions` for the types but I guess this is a trade-off we will have to accept here.

Other solution would be be more specific and use the solution suggested here https://github.com/ChainSafe/lodestar/issues/5201#issuecomment-1442394412

Reply:
That seems fine. We already using namespaced types on few other places.